spec/features/events_spec.rb

```diff
@@ -22,6 +22,49 @@
     expect(page).to have_css("div#occupied_places")
   end
 
+  scenario "logged in as Organizer I want to be unable to send emails if there is any unclassified application left" do
+    login(:organizer)
+    @event.update!(max_participants: 1)
+    @pupil1 = FactoryGirl.create(:profile)
+    @pupil1.user.role = :pupil
+    @nilApplication = FactoryGirl.create(:application_letter, :event => @event, :user => @pupil1.user, :status => 2)
+    visit event_path(@event)
+    expect(page).to have_button("Zusagen verschicken", disabled: true)
+    expect(page).to have_button("Absagen verschicken", disabled: true)
+  end
+
+  scenario "logged in as Organizer I want to be unable to send emails if there is a negative number of free places left" do
+    login(:organizer)
+    @event.update!(max_participants: 1)
+    @pupil1 = FactoryGirl.create(:profile)
+    @pupil1.user.role = :pupil
+    @pupil2 = FactoryGirl.create(:profile)
+    @pupil2.user.role = :pupil
+    @acceptedApplication = FactoryGirl.create(:application_letter, :event => @event, :user => @pupil1.user, :status => 1)
+    @acceptedApplication2 = FactoryGirl.create(:application_letter, :event => @event, :user => @pupil2.user, :status => 1)
```

Comment:
`:status => :accepted` rsp. `status: :accepted` (same for many other positions). In this case you can even use the [application_letter_accepted factory](https://github.com/hpi-swt2/workshop-portal/blob/dev/spec/factories/application_letters.rb#L29)